index.org

```diff
@@ -270,7 +271,7 @@ For example, when executing =cargo rtic-scope --bin blinky [options]=:
    For example, =device = stm32f4::stm32f401=, =binds = SysTick=, and =binds = EXTI1= might be extracted.
    Here, each IRQ label is associated with the RTIC task it is bound to.
 
-   This parsing step places some restrictions on how the source code for an RTIC application can be written. Refer to [[RTIC application constrains]].
+   This parsing step places some restrictions on how the source code for an RTIC application can be written. Refer to [[RTIC application constraints]].
 3. An adhoc cdylib crate is then built which translates IRQ labels to IRQ numbers.
```

Comment:
What is a cdylib crate?

Reply:
Rust for shared objects (.so-files). Probably better to just say "a shared object file".